lib/node_modules/@stdlib/string/base/slice/README.md

```diff
@@ -0,0 +1,96 @@
+<!--
+
+@license Apache-2.0
+
+Copyright (c) 2025 The Stdlib Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+-->
+
+# slice
+
+> Slice UTF-16 code units from a string.
+
+<section class="usage">
+
+## Usage
+
+```javascript
+var slice = require( '@stdlib/string/base/slice' );
+```
+
+#### slice( str, start, end )
+
+Slices UTF-16 code units from a string.
+
+```javascript
+var out = slice( 'last man standing', 1, 17 );
+// returns 'ast man standing'
+
+out = slice( 'Hidden Treasures', 0, 6 );
+// returns 'Hidden'
+
+out = slice( 'foo bar', 2, 7 );
+// returns 'o bar'
+
+out = slice( 'foo bar', -1, 7 );
+// returns 'r'
+```
+
```

Comment:
@headlessNode For clarity, would you also mind documenting the parameters here? In particular, it is not immediately obvious without studying the code that `start` is inclusive and `end` exclusive.